x/programs/rust/examples/automated-market-maker/src/lib.rs

```diff
@@ -22,8 +22,8 @@ const MAX_GAS: Gas = 10000000;
 
 /// Initializes the pool with the two tokens and the liquidity token
 #[public]
-pub fn init(context: &mut Context, token_x: Program, token_y: Program, liquidity_token: Id) {
-    let lt_program = context.deploy(liquidity_token, &[0, 1]);
+pub fn init(context: &mut Context, token_x: Program, token_y: Program, liquidity_token: Vec<u8>) {
```

Comment:
We should definitely still have a rust type that represents an Id. Could even be a type alias for now

`pub type Id = Vec<u8>`

Reply:
done